Makefile

```diff
@@ -24,11 +24,13 @@ database: composer
 infection: composer database
 	vendor/bin/infection
 
-integration: composer database
-	if [ ! -f "config/testing.yml" ]; then cp config/testing.yml.dist config/testing.yml; fi
```

Comment:
How about removing the `test-env` target and instead moving this line to the `database` target instead?

Reply:
I had considered that, but i feel that setting up the test config file should not be explicitly part of `make database`.

Reply:
I'm fine with it as-is. It's all going to be reconsidered for #1019 anyway.